app/main.py

```diff
@@ -2,4 +2,61 @@ class Cargo:
     def __init__(self, weight: int) -> None:
         self.weight = weight
 
-# write your code here
+
+class BaseRobot:
+    def __init__(self, name: str, weight: int, coords: list = None) -> None:
+        self.name = name
+        self.weight = weight
+        if coords is None:
+            self.coords = [0, 0]
+        else:
+            self.coords = coords
```

Comment:
Avoid using multi-line if condition, you could use either
```suggestion
        self.coords = coords if coords else [0, 0]
```
or
```suggestion
        self.coords = coords or [0, 0]
```